src/components/__tests__/DropdownFilter.spec.ts

```diff
@@ -0,0 +1,129 @@
+import { ReactElement, createElement } from "react";
+import { shallow } from "enzyme";
+
+import { DropdownFilter, DropdownFilterProps, DropdownType } from "../DropdownFilter";
+
+describe("DropdownFilter", () => {
+    const renderDropdownFilter = (props: DropdownFilterProps) => shallow(createElement(DropdownFilter, props));
+    const dropdownFilterProps: DropdownFilterProps = {
+        defaultFilterIndex: 1,
+        enableEmptyFilter: true,
+        filters: [ {
+            attribute: "Code",
+            attributeValue: "256",
+            caption: "Country",
+            constraint: "",
+            filterBy: "attribute",
+            isDefault: false
+        } ],
+        handleChange: jasmine.any(Function) as any,
+        placeholder: "Select..."
+    };
+    const optionAttributes: DropdownType = {
+        disabled: true,
+        key: "1",
+        label: "Select...",
+        value: "0"
+    };
+    const options: ReactElement<{}>[] = [
+        createElement("option", optionAttributes),
+        createElement("option", optionAttributes),
+        createElement("option", optionAttributes)
+     ];
+
+    it("renders the structure correctly", () => {
+        const dropdownFilter = renderDropdownFilter(dropdownFilterProps);
+
+        expect(dropdownFilter).toBeElement(
+            createElement("select", {
+                className: "form-control",
+                onChange: jasmine.any(Function) as any,
+                value: "1"
+            }, options)
+            );
+    });
+
+    describe("select", () => {
+        it("changes value", (done) => {
```

Comment:
This seems like you're testing the select component, which doesn't seem right. You can always expect them to behave as they were built.
Looking at your dropdown filter, think it'd have two test cases. Renders structure, and responds to changes.